samusrgrp.c

```diff
@@ -260,7 +382,6 @@ int main(int argc, char **argv)
     H_SAM = 0;
   }
 
-
```

Comment:
Once again.
Check if it makes sense to remove this line.
If it makes sense you need to put this into the 'Typos' commit.